job/client/src/main/java/alluxio/client/job/RetryHandlingJobMasterClient.java

```diff
@@ -65,13 +61,6 @@ protected long getServiceVersion() {
     return Constants.JOB_MASTER_CLIENT_SERVICE_VERSION;
   }
 
-  @Override
-  protected void beforeConnect()
```

Comment:
I assume this is a code we don't want if alluxio client can end up using this as part of adding metrics.

I have a different branch that I was partially working on where instead of doing this, I create a different service. Should I be doing that instead?

Reply:
You can continue using this one, we generally make a distinction between clients and workers/masters. In this case both the CLI and alluxio master act as clients.